src/mainboard/protectli/vault_cml/devicetree.cb

```diff
@@ -53,10 +53,10 @@ chip soc/intel/cannonlake
 	register "PchPmSlpSusMinAssert" = "2"	# 500ms
 	register "PchPmSlpAMinAssert"   = "4"	# 2s
 
-	register "tcc_offset" = "5" # TCC of 95C	
-	
+	register "tcc_offset" = "5" # TCC of 95C
```

Comment:
@macpijan  Any requests about this one? I recall Protectli wanted 75 degrees last time (but for FW6?)

Reply:
For FW6 the expectation was to match the settings from the stock FW - so the 75 deg, Maybe as a standard we should follow the same path, by checking how is it set on the stock FW first.

Reply:
@mkopec would you check?

Reply:
from turbostat on vendor firmware:

```
MSR_IA32_TEMPERATURE_TARGET: 0x05640000
```

so 5 degrees is correct.